2019-08-22 17:14:56,759 - E:\PycharmProjects\mytest\lib\parse_response.py[line:28] - DEBUG: key是error_code，运算的代码是3007 == 3006
2019-08-22 17:14:56,766 - E:\PycharmProjects\mytest\lib\util.py[line:24] - DEBUG: 现在开始写报告了：文件名是 E:\PycharmProjects\mytest\cases\test-处女座测试用例.xlsx  内容：[["{'username': 'niuhanyang', 'passwd': 'aA123456'}", "{'error_code': 0, 'login_info': {'login_time': '20190822171456', 'sign': 'fe3b890b7e19cbd157d487443b7778ec', 'userId': 9786}}", '都通过啦', '通过'], ["{'username': '15654209451', 'passwd': '123456'}", "{'error_code': 3007, 'msg': '用户名/密码错误！'}", '都通过啦', '通过'], ["{'username': '13171392458', 'passwd': '123456'}", "{'error_code': 3007, 'msg': '用户名/密码错误！'}", 'key是error_code，运算的代码是3007 == 3006', '失败']]
2019-08-22 17:14:56,790 - E:\PycharmProjects\mytest\lib\util.py[line:37] - DEBUG: 报告生成完成，文件名是E:\PycharmProjects\mytest\report\20190822171456_test-处女座测试用例.xls
2019-08-22 17:14:57,831 - E:\PycharmProjects\mytest\lib\parse_response.py[line:28] - DEBUG: key是error_code，运算的代码是3007 == 3006
2019-08-22 17:14:57,831 - E:\PycharmProjects\mytest\lib\util.py[line:24] - DEBUG: 现在开始写报告了：文件名是 E:\PycharmProjects\mytest\cases\test-处女座测试用例2.xlsx  内容：[["{'username': 'niuhanyang', 'passwd': 'aA123456'}", "{'error_code': 0, 'login_info': {'login_time': '20190822171457', 'sign': 'fe3b890b7e19cbd157d487443b7778ec', 'userId': 9786}}", '都通过啦', '通过'], ["{'username': '13930121717', 'passwd': '123456'}", "{'error_code': 3007, 'msg': '用户名/密码错误！'}", '都通过啦', '通过'], ["{'username': '18534070431', 'passwd': '123456'}", "{'error_code': 3007, 'msg': '用户名/密码错误！'}", 'key是error_code，运算的代码是3007 == 3006', '失败']]
2019-08-22 17:14:57,842 - E:\PycharmProjects\mytest\lib\util.py[line:37] - DEBUG: 报告生成完成，文件名是E:\PycharmProjects\mytest\report\20190822171457_test-处女座测试用例2.xls
2019-08-22 17:14:57,843 - E:\PycharmProjects\mytest\lib\util.py[line:42] - DEBUG: 开始发送邮件
2019-08-22 17:14:59,361 - E:\PycharmProjects\mytest\lib\util.py[line:62] - INFO: 发送邮件成功
2019-08-22 17:14:59,373 - E:/PycharmProjects/mytest/bin/start.py[line:44] - DEBUG: 运行完成
2019-08-22 17:30:09,965 - E:\PycharmProjects\mytest\lib\parse_response.py[line:28] - DEBUG: key是error_code，运算的代码是3007 == 3006
2019-08-22 17:30:09,966 - E:\PycharmProjects\mytest\lib\util.py[line:24] - DEBUG: 现在开始写报告了：文件名是 E:\PycharmProjects\mytest\cases\test-处女座测试用例.xlsx  内容：[["{'username': 'niuhanyang', 'passwd': 'aA123456'}", "{'error_code': 0, 'login_info': {'login_time': '20190822173009', 'sign': 'fe3b890b7e19cbd157d487443b7778ec', 'userId': 9786}}", '都通过啦', '通过'], ["{'username': '18273867386', 'passwd': '123456'}", "{'error_code': 3007, 'msg': '用户名/密码错误！'}", '都通过啦', '通过'], ["{'username': '13688742189', 'passwd': '123456'}", "{'error_code': 3007, 'msg': '用户名/密码错误！'}", 'key是error_code，运算的代码是3007 == 3006', '失败']]
2019-08-22 17:30:09,985 - E:\PycharmProjects\mytest\lib\util.py[line:37] - DEBUG: 报告生成完成，文件名是E:\PycharmProjects\mytest\report\20190822173009_test-处女座测试用例.xls
2019-08-22 17:30:11,087 - E:\PycharmProjects\mytest\lib\parse_response.py[line:28] - DEBUG: key是error_code，运算的代码是3007 == 3006
2019-08-22 17:30:11,087 - E:\PycharmProjects\mytest\lib\util.py[line:24] - DEBUG: 现在开始写报告了：文件名是 E:\PycharmProjects\mytest\cases\test-处女座测试用例2.xlsx  内容：[["{'username': 'niuhanyang', 'passwd': 'aA123456'}", "{'error_code': 0, 'login_info': {'login_time': '20190822173010', 'sign': 'fe3b890b7e19cbd157d487443b7778ec', 'userId': 9786}}", '都通过啦', '通过'], ["{'username': '13211751185', 'passwd': '123456'}", "{'error_code': 3007, 'msg': '用户名/密码错误！'}", '都通过啦', '通过'], ["{'username': '14582848133', 'passwd': '123456'}", "{'error_code': 3007, 'msg': '用户名/密码错误！'}", 'key是error_code，运算的代码是3007 == 3006', '失败']]
2019-08-22 17:30:11,105 - E:\PycharmProjects\mytest\lib\util.py[line:37] - DEBUG: 报告生成完成，文件名是E:\PycharmProjects\mytest\report\20190822173011_test-处女座测试用例2.xls
2019-08-22 17:30:11,106 - E:\PycharmProjects\mytest\lib\util.py[line:42] - DEBUG: 开始发送邮件
2019-08-22 17:30:13,756 - E:\PycharmProjects\mytest\lib\util.py[line:62] - INFO: 发送邮件成功
2019-08-22 17:30:13,856 - E:/PycharmProjects/mytest/bin/start.py[line:44] - DEBUG: 运行完成
